debian/snapd.install

```diff
@@ -0,0 +1,12 @@
+etc/apparmor.d/usr.lib.snapd.snap-confine
```

Comment:
If we go with the binary `snap-confine` package, this can just do into `debian/snap-confine.install`.